packages/auth/src/context/tenant/delete.ts

```diff
@@ -0,0 +1,38 @@
+import type { Tenant } from '../../contract/DbType.js';
```

Comment:
This file is already part of the `main` branch. If this is showing in the PR, then it means, the merge from `main` branch was not proper.